pagopa/pn-frontend
# Big refactor CDN OneTrust handling

## Short description
Refactored CDN OneTrust handling by using env variables

## List of changes proposed in this pull request
- Added compileOneTrustPath function in pn-commons utilities subpath
- Added new ENV variables

## How to test
First off you need to add new variables in your .env.development.local. See below:

## pn-landing-webapp
NEXT_PUBLIC_ONE_TRUST_DRAFT_MODE = ''
NEXT_PUBLIC_ONE_TRUST_PP = b5c8e1dc-89df-4ec2-a02d-1c0f55fac052

## pn-pa-webapp
REACT_APP_ONE_TRUST_DRAFT_MODE = draft
REACT_APP_ONE_TRUST_PP = 4824a110-316e-42fd-b492-8e2b0513db70
REACT_APP_ONE_TRUST_TOS = 083fb982-149c-4241-be09-12ae67d88b66

## pn-personafisica-login
REACT_APP_ONE_TRUST_DRAFT_MODE = draft
REACT_APP_ONE_TRUST_PP = 133242e7-ad9a-45e5-a417-b7a50c746899

## pn-personafisica-webapp
REACT_APP_ONE_TRUST_DRAFT_MODE = draft
REACT_APP_ONE_TRUST_PP = 365c84c5-9329-4ec5-89f5-e53572eda132
REACT_APP_ONE_TRUST_TOS = b0da531e-8370-4373-8bd2-61ddc89e7fa6
REACT_APP_ONE_TRUST_PARTICIPATING_ENTITIES = ffb2a640-8165-4d5f-94c2-6259e21bee51

## pn-personagiuridica-webapp
REACT_APP_ONE_TRUST_DRAFT_MODE = draft
REACT_APP_ONE_TRUST_PP = 365c84c5-9329-4ec5-89f5-e53572eda132
REACT_APP_ONE_TRUST_TOS = b0da531e-8370-4373-8bd2-61ddc89e7fa6

Then open Privacy Policy, Terms of Service, Participating Entities in webapps.
